automated-comments/rust/reverse-string/solution_function_not_found.md

```diff
@@ -0,0 +1,6 @@
+The test suite for the `reverse-string` exercise expects the following function defined in the `src/lib.rs` file:
+```rust
+pub fn reverse(input: &str) -> String {
+}
+```
+Please, check that a proper function and solution file are submitted.
```

Comment:
I would remove this line altogether and be more machine-like:

```suggestion
The function was not found
```

Reply:
Perhaps more explicit:

`The required function was not found`